src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/DisableRuntimeMarshallingAttribute.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/*=============================================================================
+**
+**
+**
+** Purpose: Mark up the program to indicate various legacy or new opt-in behaviors.
+**
+**
+=============================================================================*/
+
+namespace System.Runtime.CompilerServices
+{
+    /// <summary>
+    /// Disables the built-in runtime managed/unmanaged marshalling subsystem for
```

Comment:
The public docs should use the same terminology as error messages. built-in was deleted from error messages.

Reply:
(It would be nice to use same naming internally too.)